frontend/src/lib/components/MultiSelect.svelte

```diff
@@ -3,8 +3,8 @@
 
 	export let selected: string[];
 	export let options: string[];
-	export let placeholder: string;
-	export let maxSelect: number | null;
+	export let placeholder: string = '';
+	export let maxSelect: number | null = null;
```

Comment:
null = null?

Reply:
It's a default value, so we don't always have to pass a number. When maxSelect === null, then there is no max limit.

Reply:
strange that null isn't default value for null